api/config/cron-tasks.js

```diff
@@ -1,32 +1,35 @@
-module.exports = {
-  '*/1 * * * *': ({ strapi }) => {
-    const contentTypes = [
-      'api::article.article',
-      'api::restaurant.restaurant',
-      'api::page.page',
-    ];
+// module.exports = {
```

Comment:
Why comment out and not delete? You'll need it later?

Reply:
Not really, I'll delete it